packages/components/src/components/primitives/Breadcrumb/Breadcrumb.tsx

```diff
@@ -1,36 +1,69 @@
-import { useStyleConfig } from '@chakra-ui/react';
-import { Flex } from '@components';
-import React, { FC, Fragment } from 'react';
-import { IconChevronRight, Link, Text } from '..';
-import { BreadcrumbProps, BreadcrumbStyle } from './types';
+import React, { FC } from 'react';
+import { chakra, forwardRef, StyleProps, StylesProvider, useMultiStyleConfig, useStyles } from '@chakra-ui/react';
+import { getValidChildren } from '@chakra-ui/react-utils';
+import { Flex, IconChevronRight, Link, Text, FlexProps } from '@components';
+import { BreadcrumbProps, BreadcrumbStaticMembers } from './types';
 
-export const Breadcrumb: FC<BreadcrumbProps> = ({ items, separator, ...props }) => {
-  const copy = [...items];
-  const last = copy.pop();
+const LastItem = forwardRef((props, ref) => {
+  const styles = useStyles() as {
+    lastItem: StyleProps;
+  };
+  return <chakra.span aria-current="page" ref={ref} {...props} __css={styles.lastItem} />;
```

Comment:
I use a `chakra.span` here since I had `type` conflicts with our  `Text` component, when wanting to use a breadcrumb item as a link or text depending on whether it was the last item or not.